amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/records/ExportRecordsTests.java

```diff
@@ -0,0 +1,221 @@
+
+    }
+
+
+    @DataProvider (name = "CreateRecordsInRecordFolder")
+    public Object[][] getCreatedRecordsFromRecordFolder()
+    {
+        STEP("Created a records in RM site with size varying from 4 MB to 200 MB");
+        for(int i = 4; i<200; i++){
+            createRecord("Record_"+i+"MB",i*1000000);
+            i=i+20;
+        }
+
+        return recordList.stream()
+                          .map(record -> new Object[] { record })
+                          .toArray(Object[][]::new);
+    }
+
+
+    /**
+     * Given a Root Category
+     * And an open record folder
+     * When I upload a Record with size greater than 4 MB
+     * Then I should be able to export it using APT
+     */
+    @Test (description = "Testing the Export functionality on RM site suing API for records of size >4MB",
+            priority = 1,enabled = true)
+    @AlfrescoTest (jira = "APPS-986")
+    public void ExportRecord() throws Exception
+    {
+
+        STEP("Create an electronic record in RM site with size > 4MB");
+        createRecord("Record_5MB", 5000000);
+
+        STEP("Export the created record with size greater than 4 MB and verifying the expected response code");
+        exportAPI.exportRecord(getAdminUser().getUsername(), getAdminUser().getPassword(),
+                SC_OK, record.getName());
+
+    }
+
+    @Test (description = "Testing the Export functionality on RM site using API for multiple records at once with collective size of more than 4MB",
+            priority = 2,enabled = true)
+    public void ExportRecords() throws Exception
+    {
+
+        STEP("Create multiple electronic records with collective size greater than 4MB");
+
+        createRecord("Record1", 1000000);
+        createRecord("Record2", 2000000);
+        createRecord("Record3", 3000000);
+
+        STEP("Export all the created records at once and verifying the expected response code");
+        exportAPI.exportRecords(getAdminUser().getUsername(), getAdminUser().getPassword(),
+                SC_OK, recordList);
+
+    }
+
+    @Test (description = "Testing the Export functionality on RM site using API for recordFolder having records of size >4MB",
+            priority = 3,enabled = true)
+    public void ExportRecordFolder() throws Exception
+    {
+        STEP("Create an electronic record with size greater than 4 MB");
+        createRecord("Record_5MB", 5000000);
+
+        STEP("Export the Record Folder containing uploaded record with size greater than 4 MB and verifying the expected response code");
+        exportAPI.exportRecordFolder(getAdminUser().getUsername(), getAdminUser().getPassword(),
+                SC_OK, recordFolder.getName());
+    }
+
+    @Test (description = "Testing the Export functionality on RM site using API for records from size 4MB to 200MB",
+            dataProvider = "CreateRecordsInRecordFolder",priority = 4,enabled = true)
+    public void ExportRecordMultiple(Record record) throws Exception
```

Comment:
This test is not needed.

Reply:
I have restructured the tests with a common Data Provider function.